scripts/copy_output.lua

```diff
@@ -92,5 +92,6 @@ if (env.GLUON_DEVICES or '') == '' then
 	lib.exec {'rm', '-f', dest_dir('\0')..'/\0'}
 	lib.exec({'rmdir', '-p', dest_dir('\0')}, true, '2>/dev/null')
 	mkdir(dest_dir(package_prefix))
+	lib.exec {'rm', '-rf', 'openwrt/bin/targets/'..bindir..'/packages/tmp\0'}
```

Comment:
The `\0` (`*` wildcard) seems unnecessary, or can the directory name vary?
```suggestion
	lib.exec {'rm', '-rf', 'openwrt/bin/targets/'..bindir..'/packages/tmp'}
```